src/Components/NavBar.jsx

```diff
@@ -1,7 +1,17 @@
-import { ShoppingBagIcon } from "@heroicons/react/outline";
-import React from "react";
+import { ShoppingBagIcon } from '@heroicons/react/outline';
+import React, { useEffect } from 'react';
+
+export default function NavBar({ setOpen, cart }) {
+  let qtyCart = 0;
+  const updateQtyCart = () => {
+    cart.forEach((element) => {
+      // setCartSubTotal(cartsubtotal + element.price);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏commented code